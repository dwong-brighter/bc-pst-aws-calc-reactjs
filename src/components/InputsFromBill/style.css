.InputsFromBill-container {
  border-color: black;
  border-radius: 10px;
  border-width: 5px;
  background-color: palegreen;
  display: grid;
  margin: 10px;
  padding: 10px;
}
