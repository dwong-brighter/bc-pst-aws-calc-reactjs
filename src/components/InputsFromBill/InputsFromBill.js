import React from 'react';
import ExchangeRateInput from '../ExchangeRateInput';
import CostInput from '../CostInput';

/*
Can't do the following at the moment due to
https://github.com/webpack-contrib/sass-loader/issues/206
and
https://github.com/danedavid/react-redux-boilerplate/issues/1
import style from './style.css';

And then in the code you would have
<theElement className={style.theStyle}></theElement>
*/
import './style.css';

export default function InputsFromBill (props) {
  return (
    <div className="InputsFromBill-container">
      <ExchangeRateInput name="exchangeRate" description="Exchange rate (per AWS bill)" foreignCurrencyCode="USD" />
      <CostInput name="usdTotal" description="Total cost (per AWS bill)" currencyCode="USD" />
      <CostInput name="usdGst" description="GST amount (per AWS tax bill)" currencyCode="USD" />
    </div>
  );
}
