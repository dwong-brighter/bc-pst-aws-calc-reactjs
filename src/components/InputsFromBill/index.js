import InputsFromBill from './InputsFromBill.js'

export default InputsFromBill
