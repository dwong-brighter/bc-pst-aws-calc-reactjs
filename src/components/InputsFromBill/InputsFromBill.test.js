import React from 'react'
import { shallow } from 'enzyme'

import InputsFromBill from './InputsFromBill.js'

it('renders without props', () => {
  shallow(<InputsFromBill />)
})
