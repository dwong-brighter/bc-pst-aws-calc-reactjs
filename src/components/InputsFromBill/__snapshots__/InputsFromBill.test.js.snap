// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
<Styled(MuiBox)
  className="makeStyles-container-1"
>
  <ExchangeRateInput
    description="Exchange rate (per AWS bill)"
    foreignCurrencyCode="USD"
    initialRate={4.2}
    name="exchangeRate"
    onRateChange={[Function]}
  />
  <CostInput
    currencyCode="USD"
    description="Total cost (per AWS bill)"
    initialCadRounding={0.42}
    initialMainAmount={42}
    name="usdTotal"
    onCadRoundingChange={[Function]}
    onMainAmountChange={[Function]}
  />
  <CostInput
    currencyCode="USD"
    description="GST amount (per AWS tax bill)"
    initialCadRounding={0.42}
    initialMainAmount={42}
    name="usdGst"
    onCadRoundingChange={[Function]}
    onMainAmountChange={[Function]}
  />
</Styled(MuiBox)>
`;
